### PR TITLE
Play queue stored on firebase

### PR DESCRIPTION
**Pull request by fernandojsl4**

### CODE REVIEW

File: www/app/locals/local/service.js [hunk 1]
```diff
@@ -32,16 +119,26 @@
...
+      upcomingSongs.splice(1, 1);
       event.target.playVideo();
     }
 
     function onPlayerStateChange(event) {
       if (event.data == YT.PlayerState.ENDED) {
         actualSong = actualSong + 1;
-        launchNextSong(upcomingSongs[actualSong].id, upcomingSongs[actualSong].title);
+        if (actualSong > upcomingSongs.length-5) {
+          // Queue Ended
```
Comment (emmurillo) [inline www/app/locals/local/service.js]: ??

File: www/app/locals/local/view.html [hunk 2]
```diff
@@ -8,18 +8,17 @@ <h1 class="title">Ahora suena</h1>
   <ion-content>
     <div class="list card">
       <div class="item item-avatar">
-        <img src="img/ionic.png">
-        <h2>Pretty Hate Machine</h2>
-        <p>Nine Inch Nails</p>
+        <img src="{{ vm.actualSongInfo.coverArtURL }}" ng-show = "showMusicInfo">
```
Comment (emmurillo) [inline www/app/locals/local/view.html]: use the ng-show on the wrapper (item item-avatar)

File: www/app/locals/local/service.js (hunk 1 above)
Comment (emmurillo) [inline www/app/locals/local/service.js]: This seems to be wired..

Comment (fernandojsl4) [inline www/app/locals/local/service.js]: Yeap, some of the next coming up functionality, I will just remove it and add it for another ticket

**Pull request merged**